web/static/web/css/style.css

```diff
@@ -1,9 +1,9 @@
 html, body {
-    --make-yellow: rgb(248, 200, 17);
-    --make-blue: rgb(12, 66, 106);
-    --make-turquoise: rgb(0, 235, 179);
-    --make-dark-grey: rgb(74, 75, 76);
-    --make-light-grey: rgb(220, 221, 222);
+    --make-yellow: rgb(248, 200, 17); /* #F8C811 */
+    --make-blue: rgb(12, 66, 106); /* #0C426A */
+    --make-turquoise: rgb(0, 235, 179); /* #00EBB3 */
+    --make-dark-grey: rgb(74, 75, 76); /* #4A4B4C */
+    --make-light-grey: rgb(220, 221, 222); /* #DCDDDE */
 
     --make-yellow-darker: rgb(239, 184, 14);
```

Comment:
To make it easier to search through the codebase for the hex codes 🙂 (see the commit message of 68d9559ad10e7a6c1767952cee2a539162b6c230)

Reply:
But does it make any difference if you use hex instead of rgb?

Reply:
No, so we could just as well switch the places for the `rgb()` and the hex codes, but the point of this change is to have them both on the same line, both so that it's easier to find the "translation" between the color representations, and so that one can search for only the hex code and find all places that the color is used 🙂